Formula/golangci-lint.rb

```diff
@@ -35,7 +34,8 @@ def install
 
   test do
     str_version = shell_output("#{bin}/golangci-lint --version")
-    assert_match "golangci-lint has version #{version} built from", str_version
+    go_version = Formula["go"].version
+    assert_match "golangci-lint has version #{version} built with go#{go_version}", str_version
```

Comment:
Do we want this to fail if Go updates?

Reply:
Right, I forgot that this might change when Go updates. My intention was to match the Go version used to build it, but I don't think it's really necessary.